zio/parquetio/reader.go

```diff
@@ -173,6 +218,10 @@ func convertSchema(schema []*parquet.SchemaElement) ([]parquetColumn, error) {
 			continue
 		}
 
+		// XXX translate the column name
```

Comment:
The below lines are performing an operation on the column name, would you describe what more needs to be done?